### PR TITLE
Update Known Issues documentation

### PR DESCRIPTION
**Pull request by taratatach**
- Removed dead links
  - Removed solved issues
  - Added synchronisation issues with documents starting with an
    underscore or starting or ending with a space

Please make sure the following boxes are checked:

- [x] PR is not too big
- [x] it improves UX & DX in some way
- [ ] it includes unit tests matching the implementation changes
- [ ] it includes scenarios matching a new behaviour or has been manually tested
- [x] it includes relevant documentation

### CODE REVIEW

File: KNOWN_ISSUES.md [hunk 1]
```diff
@@ -26,21 +22,18 @@ Please report any other issue you may encounter by using the app help form.
...
+  synchronized at all.
+* On MacOS, Files in directories whose name start or end with a space won't be
+  synchronized although the directories themselves will be.
 
 ## Performances & resources consumption
 
-* The app takes some time before actually syncing, may hold on until
-  there is no more activity, and uses a lot of CPU on start and when syncing.
-  [details](https://trello.com/c/IQEImXQF)
+* The app uses a lot of CPU on start and when syncing.
```
Comment (sebn) [inline KNOWN_ISSUES.md]: Isn't it still the case with chokidar?

File: KNOWN_ISSUES.md [hunk 2]
```diff
@@ -26,21 +22,18 @@ Please report any other issue you may encounter by using the app help form.
 
 **Issues** caused by lower-level bugs that will be **harder to fix or circumvent**:
 
-* Replacing a directory with a file of the same name won't work (same when
-  replacing a file with a directory of the same name).
```
Comment (sebn) [inline KNOWN_ISSUES.md]: Still unsupported:

- https://github.com/cozy-labs/cozy-desktop/blob/master/test/scenarios/replace_dir_with_file/scenario.js#L6
- https://github.com/cozy-labs/cozy-desktop/blob/master/test/scenarios/replace_file_with_dir/scenario.js#L6

File: KNOWN_ISSUES.md [hunk 3]
```diff
@@ -9,15 +9,11 @@ Please report any other issue you may encounter by using the app help form.
...
-* Changing only the case or encoding in a file or directory name, e.g. renaming
-  *my stuff* to *MY STUFF*, won't be synchronized on other devices.
-  [details](https://trello.com/c/Phc3lLEr)
 * Symlinks are not supported yet. Symlinks inside the synchronized directory
   will trigger errors and their content won't be synchronized.
 * Using a mount point as the synchronized folder can have unexpected
   consequences: in case it is not mounted when the client starts, the directory
-  will appear empty, the client will assume you deleted everything and will
-  update your Cozy accordingly, putting everything in the Cozy trash.
+  will appear missing and the client will ask you to choose another one.
```
Comment (sebn) [inline KNOWN_ISSUES.md]: I believe we don't handle an empty sync dir correctly in the atom watcher yet:

| watcher | sync dir missing | sync dir empty |
| --- | --- | --- |
| atom | ok | **ko** |
| chokidar | ok | ok |

File: KNOWN_ISSUES.md (hunk 1 above)
Comment (sebn) [reply KNOWN_ISSUES.md]: Oh, didn't see you moved it to the macOS section. My bad :)

Comment (taratatach) [reply KNOWN_ISSUES.md]: I moved it to the `MacOS` section.

File: KNOWN_ISSUES.md (hunk 2 above)
Comment (taratatach) [reply KNOWN_ISSUES.md]: My bad. I thought we had fixed that.

File: KNOWN_ISSUES.md (hunk 3 above)
Comment (taratatach) [reply KNOWN_ISSUES.md]: What would happen in this case? Would we be emptying the folder again?

**Pull request merged**